ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -868,9 +866,55 @@ export const RuntimePanel = fp.flow(
 
     return runtimeRequest;
   };
+
+  // 50 GB is the minimum GCP limit for disk size, 4000 GB is our arbitrary limit for not making a
+  // disk that is way too big and expensive ($.22 an hour)
+  const validators = {
+    selectedDiskSize: {
+      numericality: {
+        greaterThanOrEqualTo: 50,
+        lessThanOrEqualTo: 4000,
+        message: '^Disk size must be between 50 and 4000 GB'
+      }
+    }
+  };
+  // We don't clear dataproc config when we change compute type so we can't combine this with the
+  // above or else we can end up with phantom validation fails
+  const dataprocValidators = {
+    masterDiskSize: {
+      numericality: {
+        greaterThanOrEqualTo: 50,
+        lessThanOrEqualTo: 4000,
+        message: 'must be between 50 and 4000 GB'
+      }
+    },
+    workerDiskSize: {
+      numericality: {
```

Comment:
can the 3 `numericality` objects be shared and use a reference? cuts down on a little bit of repetition

Reply:
I can make a little function that takes a string and says 'string must be between 50 and 4000' for the message, that should work